Frontend/src/components/SearchResults.js

```diff
@@ -1,17 +1,23 @@
 import React from 'react'
 
-const SearchResults = ( {results} ) => {
+const SearchResults = ({ results, onStockSelected }) => {
   return (
-    <ul className='absolute top-12 border-2 w-full rounded-md h-64 overflow-y-scroll bg-white border-neutral-200'>
-        {results.map((item) =>{
-            // console.log('Key:', item["1. symbol"]); // Log the key
-            return <li key={item["1. symbol"]} className='cursor-pointer p-4 m-2 flex items-center justify-between rounded-md hover:bg-emerald-200'>
-                <span>{item["1. symbol"]}</span>
-                <span>{item["2. name"].substring(0, 25)}</span>
-            </li>
-        })}
-    </ul>
-  )
+      <ul className='absolute top-12 border-2 w-full rounded-md h-64 overflow-y-scroll bg-white border-neutral-200'>
```

Comment:
Make sense. The watch list might include an <ul> so I can see why it would be beneficial to standardize the CSS. I'm thinking of styling <ul> tags in my root app.css file, then importing that file to necessary components. Would that work?

Reply:
You could do that or have an individual css in each component folder with overarching css styles/think color/standards at a higher app level.